JM/CF/Scripts/1_Core/CommunityFramework/proto/CF_Math.c

```diff
@@ -0,0 +1,16 @@
+class CF_Math
+{
+	// https://en.wikipedia.org/wiki/Cubic_Hermite_spline 
+	static vector CubicInterp(vector p0, vector p1, vector m0, vector m1, float t)
```

Comment:
A better name would be `HermiteSpline()`/`CublicSpline()` since this is not intended for linear interpolation and the suffix `Interp` sounds odd.

Or call it `HermiteInterpolation()` if it can be used for lerps as well.